src/data/helper/sourceHelper.ts

```diff
@@ -449,8 +449,8 @@ function doGuessOrdinal(
     function detectValue(val: OptionDataValue): BeOrdinalValue {
         const beStr = isString(val);
         // Consider usage convenience, '1', '2' will be treated as "number".
-        // `isFinit('')` get `true`.
-        if (val != null && isFinite(val as number) && val !== '') {
+        // `Number('')` (or any whitespace) is `0`.
+        if (val != null && Number.isFinite(Number(val)) && val !== '') {
```

Comment:
Platform compatibility is a concern. @Ovilia 
(I'll fix it later. add a polyfill)